### ISSUE TITLE
[Question] [Weblate] How to make a link to other translated item?

### ISSUE THREAD

**Issue by idealist1508** [renyuneyun/Easer#317]
For example: 

An item `@activity_history__script` should have same translation as an item `@string/title_profile`.

How can I do that in weblate?

* Option 1: I can set value of `@activity_history__script` to `@string/title_profile`
** => Problem: It doesn't work correctly, @ is saved as \@
* Option 2: I can delete value of `@activity_history__script`
** Problem 1: It works correctly only when English has this link too.
** Problem 2: In Weblate this string is untranslated

**Comment by renyuneyun** [comment]
Hmm, I believe it's not possible to do this through Webslate, as your test has already shown -- `@` will be saved as the string `@`.

Generally when strings are not marked as `translatable="false"` in the source code (resource), they are meant to be translated, even if they look like the same -- they may have different meanings or may change in the future.
But if you believe two things should actually be the same, you could make a PR (and maybe with a discussion beforehand?... maybe a bit unnecessary...) to either
* link to the other one in the source code; or
* completely remove the second one.

**Comment by idealist1508** [comment]
> But if you believe two things should actually be the same,

I think it depends on language. In some language two strings may be the same, but in another language they must be different. So, i would prefer to duplicate the translation in weblate.

**Comment by renyuneyun** [comment]
Yes, and I actually think if it's not universally the same across languages, then it's not the same one. So "duplicating" the translation is expected.

**Comment by damoasda** [comment]
We are talking about only a few occurrences per language, see screenshot below. So I suggest not to use `@string/...` in the message files and Weblate, and to replace them by normal translations, even if they look like duplicate translations. This would fix the `\@string` related translation issues in the app. Do you agree?

![string-occurrences2](https://user-images.githubusercontent.com/46045854/77471557-c8781100-6e12-11ea-9c86-5b934fbea7f1.png)
